app/services/users/profile.rb

```diff
@@ -131,7 +133,7 @@ def vet360_contact_information
         work_phone: person.work_phone,
         temporary_phone: person.temporary_phone,
         fax_number: person.fax_number,
-        text_permission: person.text_permission
+        text_permission: contact_info_v2_enabled ? nil : person.text_permission
```

Comment:
Text permission was removed from CI API V2

Reply:
Is there anything equivalent for this?

Reply:
Nope, it was removed awhile back. The text permission service is already enabled.

Reply:
Just remove this line. I looked on the frontend, there's nothing using `text_permission` or `textPermission` except tests

Reply:
Actually I'm going to create tickets for identity backend and frontend to remove a ton of these fields from the `/user` serializer and from the frontend tests. I don't see usage of basically any of these fields on the frontend

Reply:
@RachalCassity Basically do what you want with this field, I'm gonna get my team to remove it from profile soon.

Reply:
@bosawt I will remove this line and make sure it doesn't break the current Contact Information Service.

Reply:
@bosawt Done! Testing was completed locally when the va_v3_contact_information_service flipper was turned off and on.